.gitignore

```diff
@@ -1,4 +1,28 @@
+# Ignore VSCODE Settings
 .vscode/settings.json
+
+# Ignore compiled files
 tests/enum_test
 drive_simple_encoder/drive_simple_encoder
-drive_simple_encoder/drive_simple_encoder.o
+
+# Ignore the bin
+bin/
+
+# ignore .o files
+/common/*.o
+/lcmtypes/*o
+/mobilebot/*o
+/measure_motor_params/*o
+# /drive_simple_encoder/*o
+/drive_simple_encoder/*o
+/drive_simple/*o
+/test_motos/*o
```

Comment:
Could be wrong, be i think we may be missing a period here, should be `/<folder_name>/*.o` or, if all `.o` files, then we can maybe use` **/.o`

Reply:
i think it should be fine since linux is an extensionless system, but i could be wrong

Reply:
I updated the syntax to be more consistent. Both methods work but to make it clearer I went with .o. Also re-organized git ignore for easier reading